lib/util/read_repo_ast_util.js

```diff
@@ -36,6 +36,7 @@
  */
 
 const assert   = require("chai").assert;
+const async     = require("generator-async");
```

Comment:
oh, right.  The problems is not that co() doesn't work -- it's that errors get swallowed either way, and there were some bugs.  Yay.